docs/080-breaking-changes.rst

```diff
@@ -49,6 +49,27 @@ New Restrictions
   3. Explicit conversions between literals and enums are only allowed if the literal can
      represent a value in the enum.
 
+* There are new restrictions on explicit type conversions. The conversion is only allowed when there
+  is at most one change in sign, width or type-category (``int``, ``address``, ``bytesNN``, etc.)
+  For example, the conversion ``uint16(int8)`` is disallowed since the conversion changes width (8
+  bits to 16 bits) and sign (signed integer to unsigned integer). To get the previous behaviour, use
```

Comment:
```suggestion
  bits to 16 bits) and sign (signed integer to unsigned integer).
  
  In order to do the conversion, you have to go through an intermediate type.
  In this case, it can be either ``uint8`` or ``int16``. Depending on which intermediate type
  you choose, the result will be different and this is the reason why direct conversions are disallowed.
```